Libplanet/Net/Swarm.cs

```diff
@@ -1395,7 +1395,11 @@ CancellationToken cancellationToken
                     {
                         if (recentStates.Missing)
                         {
-                            throw new InvalidStateTargetException("Fuck");
+                            throw new InvalidStateTargetException(
+                                $"The {nameof(GetRecentStates)}."
+                                + $"{nameof(GetRecentStates.TargetBlockHash)}"
+                                + $" is invalid: {request.TargetBlockHash}"
```

Comment:
How about this?

```suggestion
                                $"The peer {peer} lacks the block states for {blockHash}."
```

Reply:
In this situation, the peer doesn't have the block itself as well as block states.
```
$"The peer {peer} lacks the target block: {blockHash}."
```
Would this be better? @dahlia

Reply:
That looks better!

Reply:
Fixed it and amended the commit 33c3ab8.